src/main/resources/database/changes/release-24/changelog.yml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+
+  - changeSet:
+      id: 24-1
+      author: Amit Sinha
+      changes:
+        - sqlFile:
+            comment: active Enrolment addition
```

Comment:
It does a little bit more than just adding the active enrolment - could we get a fuller description here please?

Reply:
👍